src/App.tsx

```diff
@@ -1,11 +1,37 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { SortedType } from './types/SortType';
 
 const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [query, setQuery] = useState('');
+  const [onLoading, setOnLoading] = useState(true);
+  const [sort, setSort] = useState<SortedType>(SortedType.all);
+  let filteredTodos = [...todos];
```

Comment:
Delete this, filter method doesn't mutate an array, so you don't need to copy it.
```suggestion
```